src/IECoreHoudini/GEO_CortexPrimitive.cpp

```diff
@@ -387,26 +376,77 @@ GA_PrimitiveTypeId GEO_CortexPrimitive::typeId()
 	return m_definition->getId();
 }
 
-#if UT_MAJOR_VERSION_INT >= 14
+#if UT_MAJOR_VERSION_INT >= 16
+
+void GEO_CortexPrimitive::createCortexPrimBlock(
```

Comment:
Yep, I saw that in the plugin file. I think we should just call it `create()` though.